models/repo_list.go

```diff
@@ -193,6 +199,33 @@ func SearchRepositoryByName(opts *SearchRepoOptions) (RepositoryList, int64, err
 				accessCond = accessCond.Or(collaborateCond)
 			}
 
+			var exprCond builder.Cond
+			if DbCfg.Type == core.POSTGRES {
+				exprCond = builder.Expr("org_user.org_id = \"user\".id")
+			} else if DbCfg.Type == core.MSSQL {
+				exprCond = builder.Expr("org_user.org_id = [user].id")
+			} else {
+				exprCond = builder.Eq{"org_user.org_id": "user.id"}
+			}
+
+			visibilityCond := builder.Or(
+				builder.In("owner_id",
+					builder.Select("org_id").From("org_user").
+						LeftJoin("`user`", exprCond).
+						Where(
+							builder.And(
+								builder.Eq{"uid": opts.OwnerID},
+								builder.Eq{"visibility": structs.VisibleTypePrivate}))),
```

Comment:
Please break the line before `)`, you can place a `,` at the end of the line.

Reply:
Thanks 😄 

updated.